recipes/cpp-dump/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+#include <ostream>
+#include <vector>
+
+#include "dump.hpp"
```

Comment:
```suggestion
#include <dump.hpp>

#include <iostream>
#include <ostream>
#include <vector>
```
To verify that no standard library includes are missing in the library.